### ISSUE TITLE
ImportError: cannot import name 'basestring' from 'pypika.terms'

### ISSUE THREAD

**Issue by mengyyy** [tortoise/tortoise-orm#371]
**Describe the bug**
excpetion occue when  `from tortoise.filters import get_m2m_filters` 
python3.8 x64
Ubuntu 16.04 18.04

**To Reproduce**
1. pip3 install 'tortoise-orm >=0.16.6'
2. from tortoise.filters import get_m2m_filters

**Expected behavior**
No error

**Additional context**
```python
In [1]: from tortoise.filters import get_m2m_filters
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-4ed7453a3d83> in <module>
----> 1 from tortoise.filters import get_m2m_filters

/usr/local/lib/python3.8/dist-packages/tortoise/__init__.py in <module>
     23     OneToOneFieldInstance,
     24 )
---> 25 from tortoise.filters import get_m2m_filters
     26 from tortoise.models import Model
     27 from tortoise.queryset import QuerySet

/usr/local/lib/python3.8/dist-packages/tortoise/filters.py in <module>
      5 from pypika import Table
      6 from pypika.functions import Upper
----> 7 from pypika.terms import (
      8     BasicCriterion,
      9     Criterion,

ImportError: cannot import name 'basestring' from 'pypika.terms' (/usr/local/lib/python3.8/dist-packages/pypika/terms.py)
```

**Comment by mengyyy** [comment]
but tortoise-orm==0.16.5 has not this  problem

**Comment by lntuition** [comment]
Thanks for reporting! I'll make hotfix for this issue and will be fixed with next release

**Comment by grigi** [comment]
Oh my. This was me. `basestring` isn't really a thing anymore, a regular `str` should be perfectly fine.

**Comment by grigi** [comment]
This should be fixed in our develop branch.

Please test by installing `pip install https://github.com/tortoise/tortoise-orm/archive/develop.zip` 

Thanks @lntuition ! (it was actually him)

**Comment by Bizilizi** [comment]
That helped me. Thanks

> This should be fixed in our develop branch.
> 
> Please test by installing `pip install https://github.com/tortoise/tortoise-orm/archive/develop.zip`
> 
> Thanks @lntuition ! (it was actually him)

**Comment by grigi** [comment]
Hi I released 0.16.10 with the fix :smile:
